sonar-db/src/main/java/org/sonar/db/version/v50/FeedFileSources.java

```diff
@@ -137,15 +140,17 @@
     "f.enabled = ? " +
     "AND f.scope = 'FIL' " +
     "AND p.scope = 'PRJ' AND p.qualifier = 'TRK' " +
-    "AND fs.file_uuid IS NULL";
+    "AND fs.file_uuid IS NULL " +
+    "ORDER BY s.id";
```

Comment:
this ordering applies to snapshot, not to snapshot_sources, to achieve what you suggest we should add a second ordering (desc) `snapshot_sources.id`. That's not a bad idea.